### PR TITLE
fix(AM-SonarqubeChanges): Refactor code

### PR DESCRIPTION
**Pull request by Rahul-D78**
Signed-off-by: Rahul-D78 <rahul.kumar@aveshasystems.com>

### CODE REVIEW

File: server/netop_handler.go [hunk 1]
```diff
@@ -73,22 +73,42 @@ func BootstrapNetOpPod() error {
 	return nil
 }
 
+func TcCmdError(tcCmd string, err error, cmdOut string) string {
```
Comment (gourishkb) [inline server/netop_handler.go:76]: if we are not using these util functions outside, let's not expose them.

Comment (Rahul-D78) [reply server/netop_handler.go:76]: Sure Gourish

File: server/netop_handler.go [hunk 2]
```diff
@@ -261,39 +281,41 @@ func (s *NetOps) deleteTcForSlice(sliceID string) error {
...
-				logger.GlobalLogger.Errorf(errStr)
-				return errors.New(errStr)
-			}
-			sliceInfo.tc = newTc
-			return nil
+	// Check if there are any changes in TC parameters.
+	if (*sliceInfo.tc) == (*newTc) {
+		logger.GlobalLogger.Infof("No change in Slice TC params, ignoring update")
+		return nil
+	} else {
```
Comment (gourishkb) [inline server/netop_handler.go:297]: We can simplify this further, move the else block out in normal flow. if block return ensures the rest of code will only run when the condition is false.

File: server/netop_handler.go (hunk 1 above)
Comment (Rahul-D78) [reply server/netop_handler.go:76]: Except TcCmdError rest I made them not accessible outside please review again

**Review by gourishkb — APPROVED** [review]

**Review by bharath-avesha — APPROVED** [review]

**Pull request merged**